src/modules/listItems.js

```diff
@@ -0,0 +1,90 @@
+import comments from './comment.js';
+import fetchData from './fetchAllData.js';
+import { getAPI, postAPI } from './involvementAPI.js';
+
+export const listLikes = () => {
+  getAPI().then((response) => {
+    const like = Array.from(document.querySelectorAll('.likes'));
```

Comment:
- [ ] This code functionality is working correctly although for this project you are required to use the `asyn/await` syntax instead of using the `.then` syntax. please kindly make this change.

Reply:
Thanks solved

Reply:
used async and await functions and it's working thanks